### DISCUSSION TITLE
Seeking Help in regards Save state

### DISCUSSION THREAD

**Discussion by ZahierRei** [Vdauphin/HeartsAndMinds#1631] [Question and answer]
Hi so recently after playing and then save game . When we wanted to play again it gives a warning then it cant find the cfg marker 
so when we try to enter our roles it turn us into birds .
is there a way for me to locate where it save the states to delete to start a fresh ?

**Comment by Vdauphin** [comment]
Hello,
https://github.com/Vdauphin/HeartsAndMinds/discussions/1629#discussioncomment-10087905
Server side
Cheers
